src/app/resources/resources.component.html

```diff
@@ -0,0 +1,9 @@
+<form (ngSubmit)="onSubmit(resourcesForm.value)" #resourcesForm="ngForm">
+  File Upload : <input  (change)="onChange($event)" type="file" /> 
+</form>
+<div>
+  {{message}}
+  <ol>
+    <li *ngFor="let attachment of attachments">{{_.keys(attachment.doc._attachments)[0]}}</li>
```

Comment:
This looks like the only place you are using Underscore.  The resource list won't be listing out attachments, so I don't think we need to add an entire library for this one function that we will be removing in the future.